Submissions/Yolo86/Task_1/csv2json.json

```diff
@@ -0,0 +1,18780 @@
+{
+    "movieId": "1",
+    "title": "Toy Story (1995)",
+    "genres": "Adventure|Animation|Children|Comedy|Fantasy"
+}
+{
```

Comment:
This is not the correct json format. Please use a dictionary to store all the rows containing comedy movies and then dump that dictionary into a json file.